.rubocop.yml

```diff
@@ -168,6 +168,10 @@ Style/BlockDelimiters:
   Exclude:
     - 'spec/helpers/scsb_items_helper_spec.rb'
 
+Style/GuardClause:
+  Exclude:
+    - 'app/models/search_builder.rb'
+
```

Comment:
I added this because I wanted an explicit check for whether the format param was nil, which indicates that an html page is requested